react/features/base/participants/middleware.js

```diff
@@ -130,7 +136,13 @@ MiddlewareRegistry.register(store => next => action => {
         return _participantJoinedOrUpdated(store, next, action);
     }
 
-    return next(action);
+    const result = next(action);
+
+    if (type === SET_LOCATION_URL) {
```

Comment:
Please don't add if checks if there is already a switch-case, just add another `case`!

Reply:
This cannot go into the switch because I need the action to be executed. I think you've missed `const result = next(action);` maybe?